Vision/Vision.bs

```diff
@@ -23,28 +23,15 @@ Status Text:
 Boilerplate: omit conformance
 </pre>
 
-# Intention # {#intention}
+# Purpose of this Document # {#purpose}
 
 	This document is an articulation
-	of W3C’s mission, values, purpose, and principles;
+	of W3C’s mission, its vision of the Web, and its organizational principles;
 	in other words, our vision for W3C as an organization
 	in the context of our vision for the Web itself.
 	The goal of this vision is not to predict the future,
 	but to define shared principles to guide our decisions.
 
-	This document is not intended to be
```

Comment:
This section is unnecessary.

Reply:
I think it was useful, but it may indeed no longer be necessary.